llvd/app.py

```diff
@@ -139,40 +137,36 @@ def download_entire_course(self):
                                              ['selectedVideo']['durationInSeconds']) * 1000
 
                         click.echo(
-                            click.style(f"current: {count}", fg="red"))
-                        click.echo(
-                            click.style(f"format: {self.video_format}p", fg="red"))
+                            click.style(f"\nCurrent: {chapters_index:0=2d}-{clean_name(chapter_name)}/"\
+                                f"{video_index:0=2d}-{video_name}.mp4 @{self.video_format}p"))
                         current_files = []
-                        for file in os.listdir(f"./{self.course_slug}/{clean_name(chapter_name)}"):
+                        for file in os.listdir(chapter_path):
                             if file.endswith(".mp4") and "-" in file:
                                 ff = re.split(
                                     "\d+-", file)[1].replace(".mp4", "")
                                 current_files.append(ff)
                     except Exception as e:
                         if 'url' in str(e):
                             click.echo(
-                                click.style(f"This video is locked, you probably need a premium account", fg="red"))
+                                click.style(f"This video is locked, you probably"\
```

Comment:
Add a space between `probably` and `need`

Reply:
Done